### PR TITLE
feat: add pre-comit check check-title-initial

### LINKED ISSUE #7: 标题如为英文开头，则首字母需要大写
一个正则匹配`^# [a-z]`应该就行

### PR DESCRIPTION
**Pull request by CodeChenL**
* fixes https://github.com/radxa-docs/pre-commit-hooks/issues/7

### CODE REVIEW

**Comment by CodeChenL** [comment]
@RadxaYuntian It‘s ready for review

File: pre_commit_hooks/check-title-initial.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    local ret=0
+
+    # Check if the first letter of the title is capitalized
+    if grep -qE '^# [a-z]' "$1" >/dev/null
+    then
+        ret=1
+        echo "$1: First letter of the title should be capitalized" >&2
+        if ! $DRY_RUN
+        then
+            sed -i -E 's/^# ([a-zA-Z])/# \U\1/' "$1"
```
Comment (RadxaYuntian) [inline pre_commit_hooks/check-title-initial.sh]: 这个A-Z是不是过度匹配了？

**Comment by RadxaYuntian** [comment]
另外这个pr合并了就不用去documentation那边修改

**Review by RadxaYuntian — COMMENTED** [review]
squash

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**